src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { configureStore, combineReducers } from '@reduxjs/toolkit';
+import reducerCategories from './categories/categories';
+import reducerBook from './books/books';
```

Comment:
- [ ] [OPTIONAL] Please consider installing the redux-logger npm package and importing applyMiddleware to use it. It will help you a lot in the development process 👍